src/프런터.js

```diff
@@ -0,0 +1,44 @@
+function solution(priorities, location) {
+  const stack = [];
+  let counter = 0;
+
+  for (let i = 0; i < priorities.length; i++) {
+    const printerObj = {};
+    const currentDocument = priorities[i];
+
+    printerObj[i] = currentDocument;
+    stack.push(printerObj);
+  }
```

Comment:
해당 로직을 Array 'map'을 사용하면 더 쉽고 선언적으로 작성할 수 있을 꺼 같습니다!
```suggestion
  const stack = priorities.map((currentDocument, i) => ({ [i]: currentDocument }));
```

Reply:
와.. 대박입니다 ㅋㅋㅋ 감사합니다